subt/artifacts.py

```diff
@@ -129,6 +129,31 @@ def count_yellow(img):
     return count_mask(mask)
 
 
+def count_orange_blue(img):  # for phone
+    b = img[:,:,0]
+    g = img[:,:,1]
+    r = img[:,:,2]
+    mask_blue = np.logical_and(b > 40, np.logical_and(b/1.5 > g, b/2 > r))
+
+    blue = count_mask(mask_blue)
+    count, w, h, x_min, x_max = blue
+    if count == 0 or w > 20 or h > 20:
+        return 0, None, None, None, None
+#    print(count)
+    mask_orange = np.logical_and(r > 40, np.logical_and(r/1.5 > g, r/2 > b))
+    orange = count_mask(mask_orange)
+
+#    not_mask = np.logical_not(mask_orange)
+#    img2 = img.copy()
+#    img2[not_mask] = (0, 0, 0)
+#    img2[mask_orange] = (0, 128, 255)
+#    img2[mask_blue] = (255, 0, 0)
+#    cv2.imwrite('artf.jpg', img2)
+
+    mask = np.logical_or(mask_orange, mask_blue)
+    return blue  #count_mask(mask)
```

Comment:
The "orange" stuff is probably left here by mistake? It is calculated but not used for the return value.

Reply:
Good point! I would still use combination of blue&orange but two weeks ago it looked like "blue objects" are very rare so I give it a try. There are new collected images of Phone, so here I expect another iteration.